### PR TITLE
Cicero task fixes

### PR DESCRIPTION
**Pull request by Pacman99**

### CODE REVIEW

**Review by jhbertra — APPROVED** [review]

File: nix/tullia.nix [hunk 1]
```diff
@@ -37,7 +37,9 @@ rec {
       in
       config.preset.github.status.lib.reportBulk {
         bulk.text = ''
-          echo '["x86_64-linux", "x86_64-darwin"]' | nix-systems -i
+          nix eval ${flakeUrl}#ciJobs --apply __attrNames --json | # all systems the flake declares
+          nix-systems -i | # figure out which the current machine is able to build
+          jq 'with_entries(select(.value))' # only keep those we can build
```
[the remark this answers is not in the document]
Comment (Pacman99) [reply nix/tullia.nix:42]: This doesn't actually fix anything per say, I just think its better. Although I am going to add another commit to this PR that does fix the way we're building required.

File: hydra-jobs.nix [hunk 2]
```diff
@@ -54,8 +55,13 @@ let
...
   ciJobsets = stripAttrsForHydra (filterDerivations ci);
+  ciJobsetList = lib.collect lib.isDerivation ci;
+
 in
 traceNames "" (ciJobsets // {
-  required = derivationAggregate "required-marlowe" ciJobsets;
+  required = pkgs.releaseTools.aggregate {
+    name = "required-marlowe";
+    constituents = ciJobsetList;
+  };
```
Comment (jhbertra) [inline hydra-jobs.nix:65]: :+1: tried this out locally.

**Review by jhbertra — APPROVED** [review]
Seems to do the trick.

**Comment by jhbertra** [comment]
Closing this PR for now as we are switching back to Hydra.

**Pull request closed without merging**